blog/_posts/2016-09-28-rubycats-at-codemotion-warsaw-and-euruko.md

```diff
@@ -0,0 +1,152 @@
+---
```

Comment:
please change the filename to have the correct date (same as permalink and `created_at`)